Signal/src/environment/NotificationsManager.m

```diff
@@ -155,6 +154,11 @@ - (void)notifyUserForIncomingMessage:(TSIncomingMessage *)message
     }
 
     NSString *messageDescription = message.description;
+    NSString *senderName = [contactsManager displayNameForPhoneIdentifier:message.authorId];
+    NSString *groupName = [thread.name stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
+    if (groupName.length < 1) {
+        groupName = NSLocalizedString(@"UNNAMED_GROUP_NAME", @"Named used for groups without a name.");
```

Comment:
It's quite easy to create groups without a name, so I've made sure we handle that case here.